Rakefile

```diff
@@ -4,6 +4,7 @@ require 'puppetlabs_spec_helper/rake_tasks'
 # only available if gem group releases is installed
 begin
   require 'voxpupuli/release/rake_tasks'
+  require 'puppet-strings/tasks'
```

Comment:
I am not sure if this is necessary, because this module already has an task to create the REFERENCE.md:
* https://github.com/voxpupuli/puppet-splunk/blob/862c0b312f7e2b9521a018ca8ac0454af1aed264/Rakefile#L66

Reply:
@dhoppe You were correct, removed it